playbooks/ae/setup_ae_centos.yml

```diff
@@ -0,0 +1,186 @@
+
+    - name: Create build directory for Wireshark installation
+      shell: mkdir build
+      args:
+        chdir: /home/centos
+
+    - name: Run CMAKE command for wireshark installation for AE image
+      become: yes
+      shell: cmake ../wireshark-3.2.6
+      args:
+        chdir: /home/centos/build
+
+    - name: Run MAKE command for wireshark installation for AE image
+      become: yes
+      shell: make
+      args:
+        chdir: /home/centos/build
+
+    - name: Run INSTALL command for wireshark installation for AE image
+      become: yes
+      shell: make install
+      args:
+        chdir: /home/centos/build
+
+    - name: Download and install a specific version of Python v3.8.5 for AE base image
+      shell: curl -O https://www.python.org/ftp/python/3.8.5/Python-3.8.5.tgz
+      args:
+        chdir: /home/centos
+
+    - name: Untar Python directory
+      shell: "tar -xf {{CENTOS_HOME}}/Python-3.8.5.tgz"
+
+
+    - name: Configure Python optimizations for Python 3.8 install
+      become: yes
+      shell: ./configure --enable-optimizations;
+      args:
+        chdir: /home/centos/Python-3.8.5
+
+    - name: Run INSTALL command for python 3.8 installation
+      become: yes
+      shell: make altinstall
+      args:
+        chdir: /home/centos/Python-3.8.5
+
+    - name: Remove previously existing python3 configuration
+      become: yes
+      shell: rm -f /usr/bin/python3
+      args:
+        chdir: /home/centos
+
+    - name: Reset Python3 version to Python3.8 installation
+      become: yes
+      shell: ln -s /usr/local/bin/python3.8 /usr/bin/python3
+      args:
+        chdir: /home/centos
+
+    - name: Get repo OpenDistro Elasticsearch via RPM package
+      become: yes
+      shell: curl https://d3g5vo6xdbdb9a.cloudfront.net/yum/opendistroforelasticsearch-artifacts.repo -o /etc/yum.repos.d/opendistroforelasticsearch-artifacts.repo
+      args:
+        chdir: /home/centos
+
+    - name: Install OpenDistro Elasticsearch via RPM package
+      become: yes
+      shell: yum install opendistroforelasticsearch-1.11.0-1 -y
+      args:
+        chdir: /home/centos
+
+    - name: Update links for OpenDistro Elasticsearch installation
+      become: yes
+      shell: ln -s /usr/lib/jvm/java-1.8.0/lib/tools.jar /usr/share/elasticsearch/lib/
+      args:
+        chdir: /home/centos
+
+    - name: Remove opendistro_security plugin for current OpenDistro Elasticsearch installation
+      become: yes
+      shell:  bin/elasticsearch-plugin remove opendistro_security
+      args:
+        chdir: /usr/share/elasticsearch/
+
+    - name: Install OpenDistro Elasticsearch Kibana via RPM
+      become: yes
+      shell: yum install opendistroforelasticsearch-kibana -y
+      args:
+        chdir: /home/centos
+
+    - name: Update OpenDistro Elasticsearch Kibana config file
+      become: yes
+      copy:
+        src: ./templates/opendistro_kibana.sh
+        dest: /etc/kibana/kibana.yml
+        force: yes
+
```

Comment:
You should add the installation of the TPS Python dependencies here. All you need to do is copy over requirements.txt and run the equivalent command to "pip3 install -r requirements.txt"